beacon_node/execution_layer/src/lib.rs

```diff
@@ -1823,7 +1823,9 @@ impl<E: EthSpec> ExecutionLayer<E> {
 
         // Use efficient payload bodies by range method if supported.
         let capabilities = self.get_engine_capabilities(None).await?;
-        if capabilities.get_payload_bodies_by_range_v1 {
+        if capabilities.get_payload_bodies_by_range_v1
+            || capabilities.get_payload_bodies_by_range_v2
```

Comment:
This is a pre-emptive bug fix. If ELs remove support for v1 in future we still want to use v2 by calling `get_payload_bodies_by_range` below.